ui/src/api/api.ts

```diff
@@ -49,11 +49,17 @@ export async function listPapers(offset: number = 0, size: number = 25) {
  */
 export async function getPapers(s2Ids: string[]) {
```

Comment:
As written, this function doesn't need to be `async`, since it doesn't `await` anything. It might clean the code up a bit if you did, though.

Reply:
Now it does! 🧙